simd/simd-memtest/compiler.h

```diff
@@ -13,10 +13,12 @@
 #define RESTRICT restrict
 #endif
 
-#if defined(_OPENMP) && (( __STDC_VERSION__ >= 199901L ) || (__cplusplus >= 201103L ))
+#if (__STDC_VERSION__ >= 199901L) || (__cplusplus >= 201103L)
+#ifdef _OPENMP
 #define PRAGMA(x) _Pragma(#x)
+#endif
 #else
-#warning Your compiler does not support C99/C++11's _Pragma.
+#warning Your compiler does not support C99/C++11 _Pragma.
```

Comment:
The `'` breaks parsing in some editors, like VSCode (yeah it's a bug but...)

Reply:
Normally, I refuse to do anything to support Microsoft's broken compiler and related tools, but since I've removed countless `'` from NWChem comments because they break VIM's Fortran syntax highlighting, I'll allow it.

Reply:
@jeffhammond - note that I'm not using MS compiler, this is all on Linux - but I've been trying out VSCode as a new editor and it's reasonable. It was VSCode highlighting that was breaking, but since they are using the new "language server" stuff, maybe it applies to any editor usign language server? Not sure...